client/state/data-layer/wpcom/activity-log/rewind/restore-status/index.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import { pick, delay } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_RESTORE_PROGRESS_REQUEST,
+} from 'state/action-types';
+import {
+	getRewindRestoreProgress,
+	updateRewindRestoreProgress,
+	rewindRestoreUpdateError
+} from 'state/activity-log/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { exponentialBackoff } from 'state/data-layer/wpcom-http/pipeline/retry-on-failure/policies';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const debug = debugFactory( 'calypso:data-layer:activity-log:rewind:restore-status' );
+
+const requestRestoreProgress = ( { dispatch }, action ) => {
+	dispatch( http( {
+		apiVersion: '1',
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind/${ action.restoreId }/restore-status`,
+		retryPolicy: exponentialBackoff(),
```

Comment:
My inclination would be to stick with the defaults, thereby taking advantage of whatever current policy is deemed best overall Calypso. I think we get that by not specifying `retryPolicy` at all? I don't feel strongly either way.